docs/flink_job_cluster_guide.md

```diff
@@ -0,0 +1,108 @@
+
+## Prerequisites
+
+
+- Fink Operator Image Version:  ```gcr.io/flink-operator/flink-operator:v1beta1-6``` follow these [instructions](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/tree/master/helm-chart/flink-operator) to deploy the Operator
+- Flink Image Version: ```flink:1.9.3``` or ```flink:latest```
+- Helm Version 2.x or 3.x
+
+
+## Installing the Chart
+
+The instructions to install the Flink Job Cluster chart:
+
+1. Clone the repository to your local machine, which has access to your running kubernetes cluster.
+  ```bash
+  git clone https://github.com/GoogleCloudPlatform/flink-on-k8s-operator.git
+  ```
+2. Navigate to the following folder: ```/flink-on-k8s-operator/helm-chart```
+
+3. Use the following command to install the Flink job cluster chart:
+  ```bash
+  helm template --namespace example-namespace flink-job-cluster ./flink-job-cluster -f ./flink-job-cluster/values.yaml |kubectl apply -f -
+  ```
+
+***Note*** the ```values.file``` in ```/flink-on-k8s-operator/helm-chart/flink-job-cluster/``` is just an example configuration. You can use your own values.yaml if you wish and edit the parts that you want to change. The current values.yaml has the minimum configuration requirements enabled for the Flink job cluster to start successfully. 
+
+## Uninstalling the Chart
+
+To uninstall your release:
+
+1. Navigate to the following folder: ```/flink-on-k8s-operator/helm-chart```
+
+2. Use the following command to uninstall the Flink job cluster chart:
+  ```bash
+  helm template --namespace example-namespace flink-job-cluster ./flink-job-cluster -f ./flink-job-cluster/values.yaml |kubectl delete -f -
+  ```
+
+## Check the Flink job cluster
+
+After using the helm template command, the following resources will be deployed
+
+- Flink job (1x)
+- Flink job manager (1x)
+- Flink task manager (2x)
+
+You can check the status of your deployment by using
+```bash
+kubectl get deployments --namespace=<example-namespace>
+```
+
+## Flink Operator Releases
+
+You can check which images of the Operator are available at [GoogleCloudPlatform](https://console.cloud.google.com/gcr/images/flink-operator/GLOBAL/flink-operator?gcrImageListsize=30)
+
+## Monitoring
+
+The Flink job cluster comes with a PodMonitor resource, which is the counter part to a ServiceMonitor.
+The PodMonitor will use pod labels and configure prometheus to scrape the Flink job cluster metrics. Reason for using the podMonitor is simple, the Flink job cluster does not deploy services.
+
+You can use the following [dashboard](https://grafana.com/grafana/dashboards/10369) in your grafana to monitor the flink-job-cluster.
+
+## Run Jobs with InitContainer
+
+You have the option to download blob containers, which would be your JAR files to execute as jobs, directly into the Flink job cluster pods.
```

Comment:
I don't know what you mean by "block containers", is it a term of Azure? I think we might want to avoid vendor-specific stuff here. Just remind the user that they can download job jars from the remote with init containers, but we don't need to go into vendor-specific technology.

Reply:
Yes, it is a term of Azure, at least we did do it with azure blob containers. Sure, we can leave out vendor-specific technology.